.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
what does this do?

Reply:
When yarn/npm installs they by default install to `node_modules`.   If one package has a different version of the same package, let's say `react@17` vs `react@18`.   Then the package will be at `packages/my_workspace_package/node_modules` instead of at the root level node_modules.    By default npm and yarn hoist.

Hoisting is bad and causes issues.   But npm and yarn installs would be way slower if they didn't hoist.

Pnpm doesn't have this same problem.  PNPM only installs to `node_modules/.pnmm-store` (might be different folder name forgot actual name).   It then symlinks in rest of repo.   SO it can do the better behavior of never hoisting.

Me adding shamefully-hoist to start is me keeping the behavior as close as possible to what yarn was doing before.    Makes it less likely I need to make changes or something will break.   We should try to remove it though... it is shameful

Reply:
I can't remove it though until I verify everything is working with this here.   It removes a variable of what could go wrong making anything easier to debug so it has to be last